### PR TITLE
feat: add contamination parameter for Control-FREEC

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -89,12 +89,14 @@ def helpMessage() {
                                           Requires that --ascat_ploidy is set
       --cf_coeff                    [str] Control-FREEC coefficientOfVariation
                                           Default: ${params.cf_coeff}
+      --cf_contamination_adjustment[bool] Control-FREEC contaminationAdjustment
```
Comment (drpatelh) [inline main.nf:92]:
```suggestion
      --cf_contamination_adjustment [bool] Control-FREEC contaminationAdjustment
```

Comment (drpatelh) [reply main.nf:92]: No space 😱

Comment (maxulysse) [reply main.nf:92]: Won't align if I add the space, and as I'll sync with TEMPLATE after, won't really matter anyway...

Comment (drpatelh) [reply main.nf:92]: I know. Real world problems! I was just being pedantic :)

**Review by drpatelh — APPROVED** [review]

**Pull request merged**